### ISSUE TITLE
No objects were detected.

### ISSUE THREAD

**Issue by chensers** [calmisential/TensorFlow2.0_SSD#10]
Hi, i just run train with 55 pictures, and the last epoch is show this
`Epoch: 49/50, step: 6/7.0, time spent: 2.40s, loss: 2.53578, cls loss: 4.89513, reg loss: 0.17644`
but i use the one in train 55 pictures to test, it's also no objects were detected.
why, can you share a model you trained for me? Thanks

**Comment by raduciou** [comment]
got the same problem. training on 1000 pics and for 30 epochs.

**Comment by JosephTeja** [comment]
Me too...trained model for 17k pictures and unable to detect the objects. @raduciou @chensers  are you guys able to solve this?

**Comment by bamboo98** [comment]
Epoch: 35/36, step: 37/38.0, 6.54s/step, loss: 1.81877, cls loss: 3.55806, reg loss: 0.07948
same

**Comment by jianhaifeng** [comment]
same to me.No objects were detected.

**Comment by MaxxTr** [comment]
Same to me... Looks like there is some issue with detections reshape...
